content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
++++
+type = "article"
+title = "R1/beta2 – Release Notes"
+tags = []
+date = "2020-05-24 11:43:16+01:00"
+draft = true
++++
+
+The second beta for Haiku R1 marks twenty months of hard work to improve Haiku's hardware support and its overall stability. Since beta 1, there have been 101 code contributors with almost 2800 code commits. More than 900 bugs and enhancement tickets have been closed for this release. 
+
+Please keep in mind that this is beta-quality software, which means it is feature complete but still contains known and unknown bugs. While we are mostly confident in its stability, we cannot provide assurances against data loss.
+
+*To download Haiku, see "[Get Haiku!](/get-haiku/)". You may also upgrade an existing Beta 1 installation using the [instructions](#upgrading-from-beta-1). For press inquiries, see "[Press contact](#press-contact)".*
+
+## Table of Contents
+ - [System requirements](#system-requirements)
+ - [Upgrading from Beta 1](#upgrading-from-beta-1)
+ - [Beta 2 Highlights](#beta-2-highlights)
+ - [Other enhancements](#other-enhancements)
+ - [Changes for Developers and Porters](#changes-for-developers-and-porters)
+ - [New Contributor](#new-contributor)
+ - [Source Code](#source-code)
+ - [Reporting Issues](#reporting-issues)
+ - [Press contact](#press-contact)
+
+## System requirements
+This release is available on the x86 32-bit platform, as well as the x86_64 platform. Note that BeOS R5 compatibility is only provided on the 32-bit images.
+
+<table><tr><td>
+<h4>MINIMUM (32-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Pentium II; AMD Athlon</li>
+<li><strong>Memory:</strong> 256MB</li>
```

Comment:
256MB seems barely enough to boot the Desktop (after a long time) and it is not possible to start even AboutSystem apparently.